config/src/main/java/org/axonframework/config/EventProcessingModule.java

```diff
@@ -77,6 +77,12 @@
 public class EventProcessingModule
         implements ModuleConfiguration, EventProcessingConfiguration, EventProcessingConfigurer {
 
+    private static final TrackingEventProcessorConfiguration DEFAULT_SAGA_TEP_CONFIG =
+            TrackingEventProcessorConfiguration.forSingleThreadedProcessing()
+                                               .andInitialTrackingToken(StreamableMessageSource::createHeadToken);
+    private static final Function<Class<?>, String> DEFAULT_PROCESSING_GROUP_FUNCTION =
```

Comment:
This default is for sagas only. I'd like to have that depicted in the constant name.

Reply:
Only fair. Adjusted it just now.